### PR TITLE
ssht: New version 1.3.4

### PR DESCRIPTION
**Pull request by eschnett**
ssht changed configuration mechanism from "home-grown" to "cmake. The previously current version 1.2b1 (a beta release) is thus unfortunately not available any more.

### CODE REVIEW

File: var/spack/repos/builtin/packages/ssht/package.py
Comment (alalazo) [inline var/spack/repos/builtin/packages/ssht/package.py]: You should be able to omit this:
```suggestion
```
since it's taken care of by the base class.

**Review by alalazo — CHANGES_REQUESTED** [review]
Just a minor comment, otherwise LGTM.

**Review by alalazo — APPROVED** [review]

**Review by adamjstewart — APPROVED** [review]
The usual discussion of "should we drop the old version or should we support multiple build systems" applies here, but @eschnett is the maintainer of this package, so I'll defer to his decision to drop the old version instead of supporting both.

**Pull request merged**
